staging/src/k8s.io/client-go/transport/round_trippers.go

```diff
@@ -556,40 +583,67 @@ func (rt *debuggingRoundTripper) RoundTrip(req *http.Request) (*http.Response, e
 
 	reqInfo.complete(response, err)
 
-	if rt.levels[DebugURLTiming] {
-		klog.Infof("%s %s %s in %d milliseconds", reqInfo.RequestVerb, reqInfo.RequestURL, reqInfo.ResponseStatus, reqInfo.Duration.Nanoseconds()/int64(time.Millisecond))
+	kvs = make([]any, 0, 10)
+	if levels&(1<<DebugURLTiming) != 0 {
+		kvs = append(kvs, "verb", reqInfo.RequestVerb, "url", reqInfo.RequestURL)
+	}
```

Comment:
did we lose reqInfo.RequestURL? I don't see that included in kvs

Reply:
Huh? It's right here in line 588.

Or did you mean status? That gets added below in line 591. Duration is in 603.

The if checks should be so that the same information is included for each constant.

Reply:
> Huh? It's right here in line 588.

🤦  apparently I'm unable to read more than two args to a function